databroker/examples/sample_data/image_and_scalar.py

```diff
@@ -13,9 +13,9 @@
 
 
 # number of motor positions to fake
-num1 = 20
```

Comment:
this is referenced in the tests.  Either update the tests or put an alias back in.